@utahdts/utah-design-system/react/components/widgetsIndicators/Badge.jsx

```diff
@@ -0,0 +1,21 @@
+import { joinClassNames } from '../../util/joinClassNames';
+
+/**
+ * @param {object} props
+ * @param {import('react').ReactNode} [props.children]
+ * @param {string} [props.className]
+ * @param {string} props.title
+ * @returns {import('react').JSX.Element}
```

Comment:
what other common props should this have from PROPTYPES.md?
- innerRef?
- ...rest?